app/src/main/java/net/programmierecke/radiodroid2/station/StationActions.java

```diff
@@ -212,6 +212,52 @@ public static void playInRadioDroid(final @NonNull Context context, final @NonNu
                 PlayerType.RADIODROID, () -> Utils.play(radioDroidApp, station));
     }
 
+    public static void playInExternalPlayer(final @NonNull Context context, final @NonNull DataRadioStation station) {
```

Comment:
There is a much more compact way to play in external player, see PlayerSelectorAdapter.java:194

Reply:
Oh yes, indeed.